gcp/shared/utils/config_models.py

```
@@ -164,6 +166,48 @@ class FollowupSurveysConfig(BaseModel):
+        ..., description="shift_label -> [HH:MM, HH:MM, HH:MM]"
+    )
+
+    @field_validator("shifts")
+    @classmethod
+    def validate_shifts(cls, v: dict[str, list[str]]) -> dict[str, list[str]]:
+        hhmm = re.compile(r"^\d{2}:\d{2}$")
+        for key, times in v.items():
+            if len(times) != 3:
+                raise ValueError(
+                    f"shift '{key}' must have exactly 3 times, got {len(times)}"
+                )
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)